app/src/main/AndroidManifest.xml

```diff
@@ -70,6 +71,7 @@
             android:name=".SignupActivity"
             android:label="@string/title_activity_login"
             android:parentActivityName=".SplashScreenActivity"
+            android:windowSoftInputMode="adjustPan|stateAlwaysHidden"
```

Comment:
Did you identify when this is not working? I put this in my manifest and now it works as intended and don't get your problem. Or is it not working at all?